TECHNICAL-WRITING-GUIDE.md

```diff
@@ -0,0 +1,155 @@
+# Kong's Technical Writing Guide
+
+### Table of Contents
+
+- [Introduction](#introduction)
+- [Style and Tone](#style-and-tone)
+- [Types of Documentation and Structure](#types-of-documentation-and-structure)
+  - [Reference Documentation](#reference-documentation)
+  - [Explanatory Documentation](#explanatory-documentation)
+  - [How-To Guides](#how-to-guides)
+  - [Tutorials](#tutorials)
+
+
+### Introduction
+
+Kong's documentation community is growing. That means there are more and more 
+people writing, commenting, editing, and otherwise contributing to Kong's 
+official documentation. But with more contributors comes more variation in
+writing style, so in order to maintain a consistent voice and style throughout
+all of Kong's documentation, we have curated a set of technical writing guidelines. 
+
+To decrease the turn around time for publishing docs we ask that all
+contributors read through this guide in its entirety, reference our 
+[Style Guide](/STYLEGUIDE.md), and consider using one of Kong's documentation
+[templates](/templates).
+
+
+## Style and Tone
+
+When writing documentation for Kong ask yourself the following questions:
+
+**1. Is it friendly?**
+
+Here at Kong we maintain an approachable yet professional tone. This means docs
+should:
+
+- be written for all experience levels
+- be written in first person plural (e.g "we")
+- avoid placing "blame" on the user
+- avoid excessive jokes and informal jargon
+
+
+**2. Is it helpful?**
+
+Will this document or update help users to understand a new concept, set up
+something from scratch, or answer a question?
+
+Writers and editors for Kong docs should strive to cover their topic throughly 
+and succinctly. The goal is to help users accomplish a task from start to finish
+or learn something rather than just copy and pasting bits of code. Documentation
+should also help guide users to other relevant documentation or resources.
+
+**3 - Is it true?**
+
+Kong documentation should be technically correct and contain enough detail to 
+provide users with the correct context. Before publishing, writers should have
+other contributors review and verify the technical accuracy of a document. This
+ensure that users will be able to follow a doc from start to finish.
+
+## Types of Documentation and Structure
+Kong docs contain four “types” of documentation
```

Comment:
I don't know if an empty space between heading and content matters, just following the pattern.

```suggestion

Kong docs contain four types of documentation
```